LICENSE-binary

```diff
@@ -226,6 +226,7 @@ org.scalanlp:breeze_2.12
 org.typelevel:macro-compat_2.12
 org.yaml:snakeyaml
 org.apache.xbean:xbean-asm5-shaded
```

Comment:
I'm pretty sure you can delete this as obsolete, while we're at it. We're on ASM7.